### PR TITLE
[Merged by Bors] - Ff rewards

### PR DESCRIPTION
**Pull request by avive**
Add mapstructure to reward config so it is configurable via base-config toml

### CODE REVIEW

**Comment by avive** [comment]
Needed so this will work: https://github.com/spacemeshos/lite-env/pull/1

**Comment by avive** [comment]
bors try

File: mesh/reward.go [hunk 1]
```diff
@@ -8,12 +8,12 @@ import (
...
 type Config struct {
 	// reward config
-	BaseReward *big.Int
+	BaseReward *big.Int `mapstructure:"base-reward"`
 }
 
 func DefaultMeshConfig() Config {
 	return Config{
-		BaseReward: big.NewInt(5000),
+		BaseReward: big.NewInt(50*10 ^ 12),
```
Comment (antonlerner) [inline mesh/reward.go]: comment on how much it is actually in smeshes

**Review by antonlerner — APPROVED** [review]

File: mesh/reward.go (hunk 1 above)
Comment (avive) [reply mesh/reward.go]: 50

**Comment by avive** [comment]
bors try

File: mesh/reward.go [hunk 2]
```diff
@@ -3,17 +3,18 @@ package mesh
...
 type Config struct {
 	// reward config
-	BaseReward *big.Int
+	BaseReward *big.Int `mapstructure:"base-reward"`
 }
 
 func DefaultMeshConfig() Config {
 	return Config{
-		BaseReward: big.NewInt(5000),
+		BaseReward: big.NewInt(50 * int64(math.Pow10(12))),
```
Comment (antonlerner) [inline mesh/reward.go:17]: please comment on how much this is in smeshes

Comment (avive) [reply mesh/reward.go:17]: 50

**Comment by avive** [comment]
@antonlerner - can you please approve & merge?

**Comment by avive** [comment]
bors merge

**Comment by noamnelke** [comment]
bors merge

**Pull request closed without merging**